lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -465,8 +468,12 @@ def main():
                 format=module.params['format'],
                 host=module.params['host'],
             )
-        elif state == 'present':
-            sd_id = storage_domains_module.create()['id']
+        elif state == 'present' or state == 'import':
+            try:
+                sd = storage_domains_service.storage_domain_service(module.params['id']).get()
```

Comment:
I believe this is not needed, when you change the code as I've suggested in the first patch.